content-location.php

```diff
@@ -85,13 +85,11 @@
 	$alertContent = cf("alert_content");
 ?>
 
-<div id="stage" class="inner row group" role="main">
```

Comment:
Changes to this file are for template standardization AND switching from svg to font-icon for the sitewide alert to fix a safari bug.